src/weather_magic/core.cljs

```diff
@@ -93,14 +104,20 @@
 
 (defn draw-frame! [t]
   (transforms/update-lat-lon)
-  (if (:play-mode (:left @state/date-atom))
-    (update-year-month-info t :left)
-    (swap! state/year-update assoc-in [:left :time-of-last-update] (* 5 t)))
-  (if (:play-mode (:right @state/date-atom))
-    (update-year-month-info t :right)
-    (swap! state/year-update assoc-in [:right :time-of-last-update] (* 5 t)))
-  (draw-in-context state/gl-ctx-left @state/camera-left @state/background-camera-left @state/base-texture-left @state/textures-left state/shaders-left :left t)
-  (draw-in-context state/gl-ctx-right @state/camera-right @state/background-camera-right @state/base-texture-right @state/textures-right state/shaders-right :right t)
+  (if (:play-mode (:year (:left @state/date-atom)))
+    (update-year-info t :left)
+    (swap! state/year-update assoc-in [:left :year :time-of-last-update] (* 5 t)))
+  (if (:play-mode (:month (:left @state/date-atom)))
+    (update-month-info t :left)
+    (swap! state/year-update assoc-in [:left :month :time-of-last-update] (* 1 t)))
+  (if (:play-mode (:year (:right @state/date-atom)))
+    (update-year-info t :right)
+    (swap! state/year-update assoc-in [:right :year :time-of-last-update] (* 5 t)))
+  (if (:play-mode (:month (:right @state/date-atom)))
+    (update-month-info t :right)
+    (swap! state/year-update assoc-in [:right :month :time-of-last-update] (* 1 t)))
+  (draw-in-context state/gl-ctx-left @state/camera-left @state/background-camera-left @state/base-texture-left @state/textures-left state/shaders-left :left :year t)
```

Comment:
Vi skickar alltid in :year som year-month-key. Kanske bra att ändra det beroende på play-mode eller?